lib/classes/Swift/Events/Event.php

```diff
@@ -24,8 +24,6 @@ public function getSource();
 
     /**
      * Prevent this Event from bubbling any further up the stack.
-     *
-     * @param bool $cancel, optional
```

Comment:
I do understand dropping mixed, but dropping bool does indeed remove information here, as one could think of bool in this case, but true is also a valid mixed value. WDYT? Should be just follow the cs-fixer results here?

Reply:
I think the issue here is that phpdoc does not provide any extra information. Optional? Indicates the sinature. Bool? Indicates the sinature. I can accept that phpdoc is useless here.

Reply:
The phpdoc is not always useless, because it is the symfony's way of indicating that the signature is about to change. This is the basis for the deprecation notices sent by one of the component. Something like that could be introduced in our projects, so that version by version we could do the BC breaks that php (or we) force.

I will find out which rule is responsible for this removal.

Reply:
LOL. The comma was the traitor. If I set `allow_unused_params: false` it keeps alone.
What happened?
It incorrectly assumes that the comma is part of the variant name. The signature does not use a variable "$cancel," so it is unused. I guess this is a bug.